_data/banking.yml

```diff
@@ -663,8 +663,9 @@ websites:
       url: https://seb.se/
       img: seb.png
       tfa: Yes
+      software: Yes
```

Comment:
And to be precise, BankID does not require Swedish citizenship. It requires Swedish permanent resicendy, ie you need to be assigned a personal number by the Swedish tax authority.